frontend/src/domains/places/components/SearchBox/SearchBox.tsx

```diff
@@ -40,42 +41,42 @@ const SearchBox = ({ onChange, handleSearchPlaceMap }: SearchBoxProps) => {
   };
 
   return (
-    <>
-      <Flex direction="row" gap={1} width="100%" css={{ position: 'relative' }}>
-        <Input
-          id="search"
-          value={keyword}
-          icon="search"
-          placeholder="장소를 검색하세요"
-          onChange={handleChangeKeyword}
-          onKeyDown={handleEnterSearch}
-          autoFocus
-        />
-        <Button
-          variant="primary"
-          width="20%"
-          type="button"
-          onClick={handleSearch}
-        >
-          <Flex width="100%">
-            <Text color={theme.colors.white} variant="label">
-              검색
-            </Text>
-          </Flex>
-        </Button>
-        <Flex
-          width="100%"
-          css={{ position: 'absolute', top: '100%', marginTop: '1rem' }}
-        >
-          {searchResults.length > 0 && (
-            <SearchList
-              searchResults={searchResults}
-              handleSelect={handleSelect}
-            />
-          )}
+    <Flex direction="row" gap={1} width="100%" css={{ position: 'relative' }}>
+      <Input
+        id="search"
+        value={keyword}
+        icon="search"
+        placeholder="장소를 검색하세요"
+        onChange={handleChangeKeyword}
+        onKeyDown={handleEnterSearch}
+        autoFocus
+      />
+      <Button
+        variant="primary"
+        width="20%"
+        type="button"
+        onClick={handleSearch}
+      >
+        <Flex width="100%">
+          <Text color={theme.colors.white} variant="label">
+            검색
+          </Text>
         </Flex>
+      </Button>
+      <Flex
+        width="100%"
+        css={{ position: 'absolute', top: '100%', marginTop: '1rem' }}
```

Comment:
```suggestion
        css={css`
          position: absolute;
          top: 100%;
          margin-top: 1rem;
        `}
```
요렇게 Emotion의 css 함수를 사용하는 것은 어떤지 제안드립니다? 순수 css 문법을 사용하기 때문에 가독성이 더 좋은 것 같고 나중에 css를 다른 파일이나 코드로 분리할 때도 수정할 것이 없어서(ex. marginTop -> margin-top) 분리하기 편리할 것 같습니다.